SalesforceCommerceCloud/pwa-kit
# @W-12627180@ Improve keyboard accessibility of product scroller

# Description

1. `(<)` `(>)` scroll left/right buttons are no longer rendered with the skeleton, since the skeleton is not interactive
2. Wishlist 💟 button now says "Add {product} to wishlist" / "Remove {product} from wishlist" instead of just "Wishlist"
3. Tab focus indicator is now fully displayed.

Before:
![product in scroller with poorly implemented focus indicator](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/62956339/b52be48e-52f5-4dbf-92b6-134a50935e1c)

(very subtle in top left / bottom left corners)

After:
![product in scroller with focus indicator fully visible](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/62956339/99706777-8f98-4e5d-9aed-d471f3301df2)





# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

- Deploy changes
- Open home page
- Tab through product scroller. Focus should be indicated by full square.
- Turn on VoiceOver. Wishlist button should read out more than just "wishlist".

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
